packages/core/directives/map.ts

```diff
@@ -530,28 +551,44 @@ export class AgmMap implements OnChanges, OnInit, OnDestroy {
 
   private _handleIdleEvent() {
     const s = this._mapsWrapper.subscribeToMapEvent<void>('idle').subscribe(
-        () => { this.idle.emit(void 0); });
+      () => {
+        this.idle.emit(void 0);
+      });
     this._observableSubscriptions.push(s);
   }
 
   private _handleMapMouseEvents() {
     interface Emitter {
       emit(value: any): void;
     }
-    type Event = {name: string, emitter: Emitter};
+
+    type Event = { name: string, emitter: Emitter };
 
     const events: Event[] = [
-      {name: 'click', emitter: this.mapClick},
-      {name: 'rightclick', emitter: this.mapRightClick},
-      {name: 'dblclick', emitter: this.mapDblClick},
+      { name: 'click', emitter: this.mapClick },
+      { name: 'rightclick', emitter: this.mapRightClick },
+      { name: 'dblclick', emitter: this.mapDblClick }
     ];
 
     events.forEach((e: Event) => {
-      const s = this._mapsWrapper.subscribeToMapEvent<{latLng: LatLng}>(e.name).subscribe(
-          (event: {latLng: LatLng}) => {
-            const value = <MouseEvent>{coords: {lat: event.latLng.lat(), lng: event.latLng.lng()}};
-            e.emitter.emit(value);
-          });
+      const s = this._mapsWrapper.subscribeToMapEvent<google.maps.MouseEvent>(e.name).subscribe(
+        (event: google.maps.MouseEvent) => {
+          let value: MouseEvent = {
+            coords: {
+              lat: event.latLng.lat(),
+              lng: event.latLng.lng()
+            },
+            placeId: null
```

Comment:
The value for `placeId` can be in-lined here without a conditional. If the Google event doesn't have that property, then it is set to `undefined`, which follows the existing pattern by Google and matches the previous behavior.
```suggestion
            placeId: (<google.maps.IconMouseEvent>event).placeId
```